src/act.drive.cpp

```diff
@@ -802,7 +805,7 @@ ACMD(do_control)
     else if (GET_OBJ_VAL(cyber, 0) == CYB_DATAJACK || (GET_OBJ_VAL(cyber, 0) == CYB_EYES && IS_SET(GET_OBJ_VAL(cyber, 3), EYE_DATAJACK)))
       jack = cyber;
 
-  if (IS_WORKING(ch) || AFF_FLAGGED(ch, AFF_PILOT)) {
```

Comment:
IS_WORKING needs to be preserved somewhere, otherwise people will be building ammo while they rig. If the message didn't mesh for you, it can have its own message.

Reply:
Good call, it should be a custom message but we can do that!